src/System Application/App/Performance Profiler/src/PerformanceProfileHelper.Codeunit.al

```diff
@@ -0,0 +1,55 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace System.Tooling;
+
+using System.Security.User;
+
+codeunit 1934 "Performance Profile Helper"
```

Comment:
names like management, helper, misc should be avoided. These becomes buckets of random stuff. We want single responsibility codeunits and, when you have those, the name should reflect what is in it... Maybe "Activity Type Impl." for the first bit?

Reply:
just noticed this is called from the Scheduled Perf Profiler Impl. so why not just keep the code there?

Reply:
@grobyns It is called multiple places not only Scheduled Perf Profiler Impl. And the word "helper" is used a few times in the system app. I do not understand what is teh problem with generic helper utility objects?

Reply:
Moved the code and renamed the codeunit